src/components/dev-hub/hero-banner.js

```diff
@@ -97,6 +97,7 @@ const HeroBanner = ({
     // Setting below to false would allow for bleed effect on bg
     shouldContainBackground = true,
     showImageOnMobile = true,
+    imageWidthOnMobile,
```

Comment:
```suggestion
    imageWidthOnMobile = false,
```

Let's give this a default prop so we can easily identify later this is supposed to be a boolean! I do really like how we extended the functionality here via a prop. So React!!